dev-embedded/u-boot-tools/u-boot-tools-2020.10.ebuild

```diff
@@ -12,59 +12,71 @@ SRC_URI="ftp://ftp.denx.de/pub/u-boot/${MY_P}.tar.bz2"
 
 LICENSE="GPL-2"
 SLOT="0"
-KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~x86"
-IUSE=""
+KEYWORDS="~amd64 ~arm ~arm64 ~ppc64 ~riscv ~x86"
+IUSE="minimal"
```

Comment:
Why did you add this? It seems like the only difference is about 800kb on my system.

Reply:
Because I have a use-case where only envtools are needed, instead of all binaries, and that size can make a difference on embedded systems